src/inputs/select.scss

```diff
@@ -64,3 +66,53 @@
     }
   }
 }
+
+@mixin iui-multi-select-tag-container {
+  position: absolute;
+  inset: 0 $iui-icons-default + $iui-l 0 $iui-sm;
+  height: 100%;
+  display: flex;
+  align-items: center;
+  gap: $iui-xs;
```

Comment:
might need fallback for flex gap

Reply:
amm, you mean add margins in children or what?

Reply:
Yup. Because `gap` support in flex is relatively new

Reply:
amm.. So I should add some fancy check, if gap is not suppored add margins? Cause margins go on children, whereas gap goes on parent.

Reply:
```suggestion
  gap: $iui-xs;
  > * + * {
    margin-left: $iui-xs;
    @supports (gap: $iui-xs) {
      margin-left: 0;
    }
  }
```

Reply:
Added.
Needed some cleanup on tags, but working now.